instructionAPI/src/syscalls.C

```diff
@@ -130,8 +130,7 @@ namespace x86 {
      *  so we'll consider it as our special case here.
      */
     if(id == e_call) {
-      std::vector<di::Operand> operands;
-      ins.getOperands(operands);
+      auto operands = ins.getExplicitOperands();
```

Comment:
Should this bet getAllOperands?

Reply:
This code is checking for the special `call gs:[0x10]` for 32-bit x86. Technically, that is the same as `call [gs * 16 + 0x10]` where the `16` is an implicit operand, but I haven't decided if I want to implement it in AST that way. I just guarded against it here.

Reply:
The next line explicitly checks  if(operands.size() != 1)  return false.
So I was just wandering if changing from the default behavior of checking against all operands to explicit operands would make a difference.
If there is only one operand for this instruction, then maybe it doesnt matter.

Reply:
Oh! Now I remember. The old decoder didn't include `gs` in the operands even though it is *definitey* one. In fact, it's mentioned in the comment on line 128. It's a good thing I put that there, so I could not read it in the future. At least old me was paying attention. /shrug

The updated AST using Capstone has both `gs` and `0x10`, but `gs` is taken to be implicit. I actually don't think that's correct, either. I'll double-check and update that and this test for the Capstone branch.